src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  display: block;
```

Comment:
Redundant. You don't need to use `block` for `body`. It's a block element already.